include/screencast_common.h

```diff
@@ -91,13 +91,15 @@ struct xdpw_screencast_instance {
 	struct xdpw_frame current_frame;
 
 	// pipewire
+	struct spa_source *event;
```

Comment:
I don't think `spa_source` should be used for this. Instead, we can use `pw_stream_events.process`: it's fired when a PipeWire buffer can be dequeued.

Reply:
Hm, this is kind of reverted in "screencast: drive screencast without pipewire events", then changed again in "screencast: use the pipewire callback process to start screencast". Please squash together these changes.

Reply:
> I don't think `spa_source` should be used for this. Instead, we can use `pw_stream_events.process`: it's fired when a PipeWire buffer can be dequeued.

`.process` will only be fired for all consumers when the producer enqueues a buffer, at least that was the state this summer when i wrote this. The introduction of the trigger_process call could have changed that.